styleguide/components/21-atoms/link/_link.scss

```diff
@@ -32,10 +32,14 @@ a {
   }
 
   // Standalone links (could be used for external links.)
-  &.stand-alone {
+  &.external-link {
```

Comment:
We can enforce this by using
a[href=^http] (or something similar)

Reply:
Is this the best idea? I think it's better if we do some sort of backend / JS check on all links to determine if it actually goes to an external link. Because f.e. if an editor enters a http:// link inside a body tekst somewhere linking to another page on this site we get incorrect styling due to this selector in the css code.

Reply:
Yes, it is.
That's how it is currently implemented on stad.gent:

https://stad.gent/gentse-kanaaldorpen-en-zone/nieuws-evenementen/sinksenkermis-sint-kruis-winkel-van-vrijdag-18-tot-en-met-maandag-21-mei

.node.view-mode-full div.body p a[href^="http://"]:not([class])::after, .node.view-mode-full div.body p a[href^="https://"]:not([class])::after, .node.view-mode-full div.body p a[href^="//"]:not([class])::after

Reply:
This has been added in the links refactor branch.